packages/angular-test-app/src/preview-examples/dropdown-quick-actions.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Siemens AG
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-example',
+  template: `
```

Comment:
use two files html and ts. See blind for example https://github.com/siemens/ix/blob/main/packages/angular-test-app/src/preview-examples/blind.ts https://github.com/siemens/ix/blob/main/packages/angular-test-app/src/preview-examples/blind.html

It is also necessary to adapt the documentation see https://raw.githubusercontent.com/siemens/ix/main/packages/documentation/docs/controls/blind.md